packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -94,11 +94,11 @@ install(FILES "${FLUTTER_ICU_DATA_FILE}" DESTINATION "${INSTALL_BUNDLE_DATA_DIR}
 install(FILES "${FLUTTER_LIBRARY}" DESTINATION "${INSTALL_BUNDLE_LIB_DIR}"
   COMPONENT Runtime)
 
-if(PLUGIN_BUNDLED_LIBRARIES)
-  install(FILES "${PLUGIN_BUNDLED_LIBRARIES}"
+foreach(bundled_library ${PLUGIN_BUNDLED_LIBRARIES})
```

Comment:
Why are you changing this?

Reply:
I ran into issues on Linux with search paths in the .

The generated paths work properly when processing libraries one by one, but does not when trying to process a group.

I poked around in the generated `flutter_app/build/linux/x64/debug/cmake_install.cmake` to see the differences between the two. But I could only spot the paths being different there, but not why.

The problem doesn't happen with the method channel `_plugin.so` there. I hypothesized that it is a target CMake knows about and so it adds the correct search path. And the other .so's are in the same path.

The problem also doesn't happen on Windows.

If any CMake wizard is around, I'd love to have an explanation. I can sent a diff of the `cmake_install.cmake` before and after.